docs/contributing/blog-and-website-contributions.md

```diff
@@ -11,50 +11,36 @@ Here are some things to keep in mind when deciding where to contribute to Gatsby
-- Commit and push to your fork.
-- Create a pull request from your branch.
-  - We recommend using a prefix of `docs`, e.g. `docs/your-change` or `docs-your-change`. ([PR example](https://github.com/gatsbyjs/gatsby/commit/9c21394add7906974dcfd22ad5dc1351a99d7ceb#diff-bf544fce773d8a5381f64c37d48d9f12))
-
-### Blog post format
-
-The following format can help you in creating your new blog content. At the top is "frontmatter": a fancy name for metadata in Markdown. The frontmatter for your post should include a title, date, singular author name (for now, we would welcome issues/PRs for this), and one or more tags. Your content will follow after the second set of dashes (`---`).
-
-```md
----
-title: "Your Great Blog Post"
-date: YYYY-MM-DD
-author: Jamie Doe
-excerpt: "Here is a helpful excerpt or brief description of this blog post."
-tags:
-  - awesome
-  - post
----
-
-Your next great blog post awaits!
-
-Include images by creating a folder for your post and including
-Markdown and image files for easy linking.
-
-![awesome example](./image.jpg)
-```
+If you'd like to contribute a post to the Gatsby blog, please review the process and guidelines outlined below and submit your
+idea for the post to our [Gatsby blog proposal form](https://airtable.com/shr3449954866i3iF)
+
+### Blog proposal submission process
+1. Complete and submit the [Gatsby blog proposal form](https://airtable.com/shr3449954866i3iF).
+2. A Gatsby team member will review your proposal and let you know if the proposal has been accepted within the next week or so.
+    - __If the post is accepted:__ A Gatsby team member will work with you on a timeline for submitting and reviewing a draft of your blog post and set a tentative publishing date.
+    - __If the post is not accepted:__ We’ll let you know if there are any alternative offers we can make (e.g. offer to retweet if you publish the piece elsewhere, suggest submitting it as an addition to a Gatsby doc, etc.). We’ll also do our best to explain why your proposal was not accepted and encourage you to revise your proposal based on that feedback and resubmit. Please don’t be discouraged from submitting another post in the future!
+
+If you have any questions about the process or your submission, please email [marketing@gatsbyjs.com](mailto:marketing@gatsbyjs.com).
+
+### Content guidelines for submitting a blog post proposal
+As a Gatsby community member, you have unique insight into the ins and outs of learning Gatsby, building with Gatsby, and contributing to Gatsby’s open source community. Contributing to the Gatsby blog is a great way to share your experiences and insights. Here are some guidelines for what kind of content is and isn’t a good fit for the Gatsby blog.
+
+Things we’re looking for in Gatsby blog content:
+- Information to help others overcome challenges you’ve faced while working with Gatsby
+- Stories about how Gatsby helped you overcome different challenges on work and personal projects
```

Comment:
Could this also be considered a case study?

Reply:
I think a case study would be under the umbrella of this kind of content, but it could also include more technically focused and/or less structured anecdotes of overcoming challenges. If we get a submission that seems more like a case study or something that should be in docs/tutorials, we'll check in with the Learning Team. If you all agree that it should be a doc, we'll let the author know and connect them with Learning.

Reply:
Sounds good to me.